components/pagination/Pagination.jsx

```diff
@@ -0,0 +1,117 @@
+/**
+ * Pagination.jsx
+ * Created by Lizzie Salita 10/16/17
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { formatNumberWithPrecision } from '../../helpers/moneyFormatter';
+import { calculatePageRange } from '../../helpers/paginationHelper';
+
+import Pager from './Pager';
+import LimitSelector from './LimitSelector';
+import GoToPage from './GoToPage';
+
+const propTypes = {
+    changePage: PropTypes.func.isRequired,
+    totalItems: PropTypes.number.isRequired,
+    currentPage: PropTypes.number,
+    pageSize: PropTypes.number,
+    resultsText: PropTypes.bool,
```

Comment:
Would like to recommend making this an enum based on some interesting points made [in this article](https://spicefactory.co/blog/2019/03/26/how-to-avoid-the-boolean-trap-when-designing-react-components/) about long term maintainability problems associated with boolean props.

Might be good to start a discussion around establishing a pattern for or against using boolean props and under what circumstances we would use them.

Maybe outside the scope of this delivery, but wondering if a better implementation could be found using an enum like so:

`resultsText: PropTypes.oneOfType([PropTypes.boolean, PropTypes.string])`

And then we could possibly use `customResultsText` as a className prop or as we currently have it such that it overrides the `resultsText` prop.